examples/visualization/styles/helpers/basic-style.html

```diff
@@ -46,11 +46,17 @@
       });
 
       const basicStyle = carto.viz.basicStyle({
-        color: '#f00',
-        size: 10,
-        strokeColor: '#00f',
-        strokeWidth: 1,
-        opacity: 1.0
+          // Color: hex, rgb or named color value.
```

Comment:
I would use this example to show the default ones, so I would avoid to set any value

Reply:
We already have examples for 'default styles' (lines.html, points.html, polygons.html). This one goes for 'custom' (but basic) styles